CHANGELOG.md

```diff
@@ -4,7 +4,19 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [20.8.1] - unreleased
+## [20.8.2] - unreleased
+
+### Added
+
+### Changed
+
+### Fixed
+
+### Removed
+
+[20.8.2]: https://github.com/greenbone/gsa/compare/v20.8.1...v20.8.2
+
```

Comment:
Should be added after the release.

Reply:
Right, sorry. Fixed it in the next commit